docs/faq.rst

```diff
@@ -1,11 +1,32 @@
 FAQs
 ====
 
-How does Mantl compare to `OpenStack Magnum <https://wiki.openstack.org/wiki/Magnum>`_?
+How are Mantl components chosen_?
+--------------------------------------------------------------------------------------
+
+A service design methodology was used to assemble Mantl components. Priority was given
+to sensing the large shifts in this technology and to defining an architecture 
+consistent with mode 1 to mode 2+ methodologies. Once the architecture was optimized 
```

Comment:
Mode 1 is traditional and sequential, emphasizing safety and accuracy. Mode 2 is exploratory and nonlinear, emphasizing agility and speed.